nest-cli.json

```diff
@@ -3,6 +3,7 @@
   "collection": "@nestjs/schematics",
   "sourceRoot": "src",
   "compilerOptions": {
-    "deleteOutDir": true
+    "assets": ["**/*.hbs"],
```

Comment:
any particular reasn for this change? is [it this](https://docs.nestjs.com/cli/monorepo#assets)?